Gems/ScriptCanvas/Code/scriptcanvasgem_headers.cmake

```diff
@@ -0,0 +1,378 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+set(FILES
```

Comment:
We should remove the the files in this cmake list from the 
`scriptcanvasgem_common_files.cmake` and `scriptcanvasgem_runtime_asset_files.cmake` files.
And then in the ScriptCanvas.Static target, add this file to `FILE_CMAKE` argument

Reply:
I'm making a similar change for LmbrCentral to address a similar issue.  I'll update this in that PR.